mne_realtime/rt_filter.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 19 13:30:38 2021
```

Comment:
as a historical practice, we typically have a commented out block that specify:
`# Author: Author_name <email_address>`